discovery-infra/test_infra/tools/connection.py

```diff
@@ -0,0 +1,17 @@
+import socket
+import logging
+
+
+def is_open(ip, port, timeout=5):
+    s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    s.settimeout(timeout)
+    try:
+        s.connect((ip, int(port)))
+        s.shutdown(socket.SHUT_RDWR)
+        logging.info("Ip %s with port %s are reachable", ip, port)
+        return True
+    except:
+        logging.info("Ip %s with port %s are not reachable", ip, port)
```

Comment:
I don't think that this utility should have logs inside. But anyway please transform logs inside the `except` clause to be of type `logging.exception` so the exception won't get lost.

Reply:
no reason for it to be logging.exception. It is on purpose info

Reply:
I think it is useful when there's an exception to log that exception.  Errors should never pass silently.